ui/shared/components/panel/variants/Predictions.jsx

```diff
@@ -38,39 +43,72 @@ const Prediction = ({ field, annotation, dangerThreshold, warningThreshold }) =>
     }
   } else {
     color = SEVERITY_MAP[value]
-    value = value.replace('_', ' ')
+    if (value) {
+      value = `${value}`.replace('_', ' ')
+    }
+  }
+
+  if (!name) {
+    name = field.replace(/_/g, ' ').toUpperCase()
```

Comment:
given that we define what the fields are named, why not just continue to use field names that transform nicely?

Reply:
we could do both?
primate_ai_score is an ES index field so hard to change now

Reply:
we don't pass through field names though, we map them to different client side field names in the `_variant_details` method in `saved_variant_api`

Reply:
that re-mapping in saved_variant_api will go away ideally. The current transformations are an artifact of having 2 different data sources (es. and mongo), but the ideal setup would be to pass elasticsearch results straight to client. I can get revert the regexp if you want though.